### ISSUE TITLE
String not being split by '/' delimiter

### ISSUE THREAD

**Issue by alexthaii** [JungWinter/split#19]
I have a string `folder/subfolder/file.yml` and I'd like to split it by `/`. When I print the result, it gives me the whole string rather then `folder`/

**Workflow Snippet**
```
- uses: jungwinter/split@v2.0.0
  id: split
  with:
    msg: ${{ steps.filter.outputs.changes_files }}
    separator: '/'

- name: Get application's name
  id: app_name
  run: |
    echo ${{ steps.split.outputs._0 }}
```

**Output Logs**
```
Run jungwinter/split@v2.0.0
  with:
    msg: folder/subfolder/file.yml
    separator: /
    maxsplit: -1

Run echo folder/subfolder/file.yml
  echo folder/subfolder/file.yml
```

**Comment by luketree1** [comment]
use seperator instead of separator.

**Comment by winterjung** [comment]
Thanks for reporting and sorry for the delay in responding!

To address the issue you reported, could you please try using either `jungwinter/split@v2` or `jungwinter/split@latest` instead of `v2.0.0`? I have been able to reproduce the issue on version `v2.0.0` (see [this run](https://github.com/JungWinter/split/actions/runs/4400786060/jobs/7706406873)), but it has been resolved in version `v2.1.0` (see [this run](https://github.com/JungWinter/split/actions/runs/4400806468/jobs/7706442256)), which is almost the same as `v2.1.1-rc1`. If you still encounter any issues, please let me know.
